janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -3749,4 +3754,169 @@ private GraphTraversal<Vertex, Vertex> getV(String propertyName, Object value, S
         }
         return traversal.has(propertyName, value);
     }
+
+    @Test
+    public void someTestThatIDoNotWantToKeep() throws Exception {
```

Comment:
Probably should now be called `testMarkDiscardedRegisteredIndex` or something like that :)

Reply:
I'm just too quick pushing commits I haven't reviewed myself...